src/random_test.cpp

```diff
@@ -54,7 +54,6 @@ static std::string get_result(std::string sql, Thd1 *thd) {
 /* return the db branch */
 static std::string db_branch() {
```

Comment:
can you modify to use method like 
/*
    Get version number for server in a form easy to test on

    SYNOPSIS
      mysql_get_server_version()
      mysql   Connection

    EXAMPLE
      4.1.0-alfa ->  40100

    NOTES
      We will ensure that a newer server always has a bigger number.

    RETURN
     Signed number > 323000
     Zero if there is no connection
  */

* find the pxb base version */
  static unsigned long pxb_base_version() {
    std::string pxb_base = MYSQL_SERVER_VERSION;
    unsigned long major = 0, minor = 0, version = 0;
    std::size_t major_p = pxb_base.find(".");
    if (major_p != std::string::npos) major = stoi(pxb_base.substr(0, major_p));

    std::size_t minor_p = pxb_base.find(".", major_p + 1);
    if (minor_p != std::string::npos)
      minor = stoi(pxb_base.substr(major_p + 1, minor_p - major_p));

    std::size_t version_p = pxb_base.find(".", minor_p + 1);
    if (version_p != std::string::npos)
      version = stoi(pxb_base.substr(minor_p + 1, version_p - minor_p));
    else
      version = stoi(pxb_base.substr(minor_p + 1));
    return major * 10000 + minor * 100 + version;
  }

Reply:
so with this. we can do something like this if version < 80000. then disable some feature for 5.7.  also and version < 80026

Reply:
Fixed as suggested